src/plots/plots.js

```diff
@@ -2012,11 +2012,8 @@ plots.doCalcdata = function(gd, traces) {
         //
         // This ensures there is a calcdata item for every trace,
         // even if cartesian logic doesn't handle it (for things like legends).
-        //
-        // Tag this artificial calc point with 'placeholder: true',
-        // to make it easier to skip over them in during the plot and hover step.
         if(!Array.isArray(cd) || !cd[0]) {
-            cd = [{x: false, y: false, placeholder: true}];
+            cd = [{x: false, y: false}];
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1564/commits/b5cd57308c60b584ba47ba93b35bb02efdd15418 

Didn't break any tests. The only place I can think of that this would impact in the legend code, but looks like it isn't making any assumptions on `x` / `y` [here](https://github.com/plotly/plotly.js/blob/master/src/components/legend/style.js).